src/core/symbology/qgsgraduatedsymbolrenderer.h

```diff
@@ -222,14 +222,65 @@ class CORE_EXPORT QgsGraduatedSymbolRenderer : public QgsFeatureRenderer
     Mode mode() const { return mMode; }
     void setMode( Mode mode ) { mMode = mode; }
 
+    /**
+     * Returns if we want to classify symmetric around a given value
+     * \since QGIS 3.4
```

Comment:
While here it says 3.4 so yes your SIP files are not updated in your PR (so Travis is complaining).
You need to run `scripts/sipify_all.sh 
` or setup the prepare-commit hook for the next time (https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#proper-formatting) It will sipify your files automatically before commiting your code.